packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-utils.tsx

```diff
@@ -0,0 +1,168 @@
+            ),
+        },
+        {
+            key: 'home_rental_agreement',
+            value: <Localize i18n_default_text='Home rental agreement: valid and current agreement.' />,
+        },
+    ],
+});
+
+type TAuthStatus = {
+    needs_poi?: boolean;
+    is_submitted?: boolean;
+};
+
+export const getPOAStatusMessages = (
+    status: typeof AUTH_STATUS_CODES[keyof typeof AUTH_STATUS_CODES],
+    auth_status?: TAuthStatus,
+    should_show_redirect_btn?: boolean
+) => {
+    const is_redirected_from_platform = isNavigationFromP2P() || isNavigationFromDerivGO();
+
+    const pendingButton = (onClick?: () => void, platform_name?: string) => {
+        if (!auth_status?.needs_poi) {
+            if (should_show_redirect_btn)
+                return (
+                    <VerificationStatusActionButton
+                        button_text={
+                            <Localize i18n_default_text='Back to {{platform_name}}' values={{ platform_name }} />
+                        }
+                        onClick={onClick}
+                    />
+                );
+            if (!is_redirected_from_platform) {
+                return (
+                    <VerificationStatusActionButton
+                        to='/'
+                        button_text={<Localize i18n_default_text='Continue trading' />}
+                    />
+                );
+            }
+        }
+        if (auth_status?.needs_poi) {
+            return (
+                <VerificationStatusActionButton
+                    to='/account/proof-of-identity'
+                    button_text={<Localize i18n_default_text='Proof of identity' />}
+                />
+            );
+        }
+        return null;
+    };
+
+    const resubmitButton = (onClick?: () => void) => (
+        <VerificationStatusActionButton button_text={<Localize i18n_default_text='Resubmit' />} onClick={onClick} />
+    );
+
+    const verifiedButton = (onClick?: () => void, platform_name?: string) => {
+        if (auth_status?.needs_poi) {
+            return (
+                <VerificationStatusActionButton
+                    to='/account/proof-of-identity'
+                    button_text={<Localize i18n_default_text='Proof of identity' />}
+                />
+            );
+        }
+        if (should_show_redirect_btn)
+            return (
+                <VerificationStatusActionButton
+                    button_text={<Localize i18n_default_text='Back to {{platform_name}}' values={{ platform_name }} />}
+                    onClick={onClick}
+                />
+            );
+        if (!is_redirected_from_platform) {
+            return (
+                <VerificationStatusActionButton
+                    to='/'
+                    button_text={<Localize i18n_default_text='Continue trading' />}
+                />
+            );
+        }
+        return null;
+    };
+
+    const titles: Record<typeof status, React.ReactElement> = {
```

Comment:
I'm not sure I understood you.
you mean, I should rename this file? maybe to config?
I believe we should keep each config in relevant folder, rather than in one folder somewhere outside.
I think we should keep separately some configs or utils in separate folder, if they are reused in a lot of places. Here this data is used only for poa, same think for poi and poo